example/lib/main.dart

```diff
@@ -46,8 +46,8 @@ class _MyAppState extends State<MyApp> {
     final String apptentiveKey;
     final String apptentiveSignature;
     if (Platform.isAndroid) {
-      apptentiveKey = "<YOUR_ANDROID_KEY>";
-      apptentiveSignature = "<YOUR_ANDROID_SIGNATURE>";
+      apptentiveKey = "ANDROID-FOXFAX-d2c49794fc25";
```

Comment:
Will want to remove this from the branch. Not sure if it's worth the effort to completely purge it from GitHub (involves a support request).

Reply:
Updated with the latest commit.